src/interfaces/IMulticall.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.5.0 <0.9.0;
```

Comment:
If I try compiling this file with 0.5.0, 0.6.0 I get the below error:

```
IMulticallTest.sol:10:24: TypeError: Data location must be "calldata" for parameter in external function, but "memory" was given.
    function aggregate(Call[] memory calls)
                       ^-----------------^
```

So looks like it needs that change + `pragma experimental ABIEncoderV2;`

Reply:
Good catch! This repo seems like a good use case for [multibuild](https://github.com/foundry-rs/foundry/pull/2447). I wish I will found the time to add that feature to Foundry some day.

Reply:
Awesome, thanks. Def would be useful, in the meantime setting up various CI build jobs [like this](https://github.com/foundry-rs/forge-std/blob/83c6e5b7bbbc865a83ed636ad32a8b7f2fc9185e/.github/workflows/ci.yml#L24-L60) is a decent workaround

Reply:
I actually have started working on something similar - a GitHub Action for multibuild.

https://github.com/paulrberg/foundry-multibuild/blob/aa52a9c69840d3ca5af990b5a53d47c34ced8495/src/multibuild.sh

Reply:
Looks like it might be private, getting a 404

Reply:
Whoops, made it public now.

Reply:
Ah nice, seems like it will be useful!